prereq_tools/base.py

```diff
@@ -1087,14 +1087,16 @@ def modify_prefix(self, comp_def, env):
         """Overwrite the prefix in cases where we may be using the default"""
         prebuilt1 = os.path.join(env.subst("$PREBUILT_PREFIX"),
                                  comp_def.name)
-        opt_name = "%s_PREBUILT" % comp_def.name.upper()
         # prebuilt2 can be None so add a default
-        prebuilt2 = self.__env.get(opt_name, "/__fake__")
+        prebuilt2 = self.__env.get('{}_PREBUILT'.format(comp_def.name.upper()),
+                                   "/__fake__")
+        prebuilt3 = self.__env.get('{}_PREFIX'.format(comp_def.name.upper()))
```

Comment:
Hrm.  This seems to have broken the `*_PREFIX`es when `USE_INSTALLED=all` finds components installed in `/usr` (i.e. by RPMs for example)  which is supposed to trigger lines 1100-1102 below.  This seems to be because this `prebuilt3` is getting set the `.../install/` prefix value, even when components are found *system install*ed, in `/usr`.